.github/pull_request_template.md

```diff
@@ -1,37 +1,48 @@
-Fixes | Closes | Resolves #
+# Description
 
-> Anything marked as optional that you didn't need to fill in, please remove it from the PR description. Choose one of the keywords above to refer to the issue this PR solves, followed by the issue number (e.g Fixes # 666). If there is no issue, remove the line. Remove this note after reading.
+>Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. 
+>**THIS WILL BE USED IN RELEASE NOTES**
 
-## Changes:
--
+Description here
 
-## Types of changes
+Fixes # (issue)
 
-What types of changes does your code introduce?
-_Put an `x` in the boxes that apply_
+## Type of change
 
-- [ ] Bugfix (non-breaking change which fixes an issue)
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] Documentation Update
-- [ ] Code style update (formatting, renaming)
-- [ ] Refactoring (no functional changes, no api changes)
-- [ ] Build related changes
-- [ ] Other (please describe): 
+- [ ] Refactoring (includes code styling changes etc)
+- [ ] This change requires a documentation update
+	- If marked - put a docs PR link here
+
+# How Has This Been Tested?
+
+>Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
 
-## Testing
-**Requires testing**
+*Short Testing Description*
 
-- [ ] Yes
-- [ ] No
+**Scenarios executed by me:**
+- [ ] Test A
+- [ ] Test B
 
-**In case you checked yes, did you write tests??**
+**Test Configuration used during dev testing**:
+* OS version: *to be filled*
+* CL client used: *to be filled*
+* etc
 
-- [ ] Yes
-- [ ] No
+**Scenarios or areas worth to be checked by QA**
 
-**Comments about testing , should you have some** (optional)
+*Fill any additional details which may help QA team to test this feature better*
 
-## Further comments (optional)
+# Checklist:

```

Comment:
The checklist is not useful in my opinion. these things all core devs should take in mind in general when making a pr, they do not need to check boxes every time for them.

Reply:
I agree. Most of the items are assumed by default.

Reply:
@rubo @smartprogrammer93 @LukaszRozmej @MarekM25 
What about changing this section from "Checklist" to something like "Reminder" or remain Checklist but without need to "check" anything?

This would maybe just help devs in a way like "Oh - forgot to add something" just by reading this section or remembering that it is here. If it helps once it would be a better outcome than not having it at all.